easybuild/tools/module_generator.py

```diff
@@ -962,7 +962,7 @@ def swap_module(self, mod_name_out, mod_name_in, guarded=True):
         :param mod_name_in: name of module to load (swap in)
         :param guarded: guard 'swap' statement, fall back to 'load' if module being swapped out is not loaded
         """
-        body = "module swap %s %s" % (mod_name_out, mod_name_in)
+        body = "module swap %s" % (mod_name_in)
```

Comment:
@welucas2 It's probably worth adding a comment above here to explain why we use single argument swap commands.

Before we didn't have a good reason to use `swap foo foo/<version>` rather than `swap foo/<version>` I think, but now we clearly do, so it's worth mentioning that I think...

I'll also check with the other module tools we support whether single argument swap works for those, I'm not sure that's covered by the test suite right now (we just do a text comparison of the generated module file in the tests currently).